var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -96,6 +97,14 @@ class Dealii(CMakePackage, CudaPackage):
             description='Compile with Slepc (only with Petsc and MPI)')
     variant('symengine', default=True,
             description='Compile with SymEngine')
+    # FIXME @9.3: enable by default
+    variant('simplex', default=False,
+            description='Compile with Simplex support')
+    # FIXME @9.3: enable by default
+    variant('taskflow',  default=False,
+            description='Compile with multi-threading via Taskflow')
```

Comment:
I must admit that I've resisted adding TaskFlow for now, because as far as I can tell we've not yet got a complete solution for it. So we don't yet know which version to restrict it to. Is that correct, or have I missed some update to the conversation?

Reply:
Would we also want to introduce a conflict between `+threads` and `+taskflow`?

Reply:
for the moment, there is no conflict and they co-exist. I think we can keep both around as long as all implementation based on tbb is replaced by an equivalent with taskflow. Then we'll remove tbb from 9.3. This has not yet happened though...

We don't have a complete a solution, but there is already a lot of functionality which is implemented with taskflow in master.